client/src/pages/dashboard/views/Barcodes.svelte

```diff
@@ -1 +1,16 @@
-<p>Barcodes!</p>
+<script lang="ts">
+    import { Office } from '~model/office';
+    import { dashboardState } from '../stores/DashboardState';
+
+    let currentOffice: Office['id'] | null = null;
+
+    // eslint-disable-next-line no-unused-expressions, prefer-destructuring
+    $: $dashboardState.currentOffice ? currentOffice = $dashboardState.currentOffice : null;

```

Comment:
Something is terribly off about the syntax here. Please double check the `=` and your use of the ternary _expression_.